### PR TITLE
User profile core data

### PR DESCRIPTION
**Pull request by yukim17**
Added function for saving user data, refactored task view

### CODE REVIEW

File: Applets/UI/UserProfile/UserProfileView.swift [hunk 1]
```diff
@@ -18,86 +18,46 @@ struct UserProfileView: View {
...
-                                    TextField("Permesso di Soggiorno", text: $viewModel.userProfile.permessoDiSoggiornoNumber.withDefault(""))
-                                } else {
-                                    Text(viewModel.userProfile.permessoDiSoggiornoNumber ?? "XXXXXXXXXXXXX")
-                                }
-                            }
-                            .redacted(reason: isEditing ? .privacy : .placeholder)
-                        }
+                        SecureTextField(
+                            placeholder: "Permesso di Soggiorno",
+                            data: $viewModel.userProfile.mobileNumber,
```
Comment (shinra-electric) [inline Applets/UI/UserProfile/UserProfileView.swift:59]: Wrong var. Should be Permesso.

**Comment by shinra-electric** [comment]
Apart from my comment, lgtm.

**Review by slavny11 — COMMENTED** [review]
impressive🧡

File: Applets/UI/UserProfile/UserProfileView.swift (hunk 1 above)
Comment (yukim17) [reply Applets/UI/UserProfile/UserProfileView.swift]: Fixed

**Pull request merged**